<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinMemberMainWindow</class>
 <widget class="QMainWindow" name="JoinMemberMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JoinMemberMainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background-color: #f2f8f3;
    font-family: &quot;Segoe UI&quot;, &quot;Malgun Gothic&quot;, sans-serif;
    border: none;
}

QWidget {
    border: none;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cde0d5;
    border-radius: 14px;
    padding: 6px 12px;
    font-size: 14px;
    min-height: 28px;
}

QLineEdit:focus {
    border: 1px solid #88c999;
    background-color: #fafffb;
}

QPushButton {
    background-color: #90caa7;
    border: none;
    border-radius: 18px;
    padding: 8px 18px;
    font-size: 14px;
    color: #ffffff;
    font-weight: bold;
    letter-spacing: 0.5px;
    min-height: 32px;
}

QPushButton:hover {
    background-color: #7ab894;
}

QLabel {
    background-color: #5f5f5f;
    color: #ffffff;
    font-size: 14px;
    border-radius: 12px;
    padding: 6px 10px;
    min-width: 50px;
    min-height: 28px;
    qproperty-alignment: AlignCenter;
    border: none;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_register">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>120</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>회원 가입</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>29</y>
      <width>229</width>
      <height>246</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::MarkdownText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_id"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string>P/W</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::MarkdownText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pw"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string>이름</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::MarkdownText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_name"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string>이메일</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::MarkdownText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_email"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string>주소</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::MarkdownText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_address"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
